SETUP/upgrade/14/20200404_normalize_project_details.php

```diff
@@ -24,44 +23,29 @@
     $projects[] = $row;
 }
 
-$stop_file = '/tmp/utf8_normalization';
-
-echo <<<EOF
-This will now start normalizing all of the project fields.
-This will likely take a while.
-To stop the process gracefully create $stop_file
-(eg: touch $stop_file) and it will stop after it
-finishes the table it is working on.
-
-EOF;
-
 echo count($projects) . " possible projects to normalize.\n";
 
 foreach($projects as $project)
 {
-    if(file_exists($stop_file))
-    {
-        echo "\n\n";
-        echo "Detected stop file ($stop_file), so stopping before starting " . $project["projectid"] . ".\n";
-        echo "Delete this file and restart to continue.\n";
-        exit();
-    }
-
-    
-    $sql = "
-        UPDATE projects SET
-        nameofwork    = '".mysqli_real_escape_string(DPDatabase::get_connection(), utf8_normalize($project["nameofwork"]))."',
-        authorsname   = '".mysqli_real_escape_string(DPDatabase::get_connection(), utf8_normalize($project["authorsname"]))."',
-        comments      = '".mysqli_real_escape_string(DPDatabase::get_connection(), utf8_normalize($project["comments"]))."',
-        extra_credits = '".mysqli_real_escape_string(DPDatabase::get_connection(), utf8_normalize($project["extra_credits"]))."'
-        WHERE projectid = '".$project["projectid"]."';
-    ";
-
+    $sql = sprintf("
+            UPDATE projects SET
```

Comment:
I know this is minor, but let's out-dent the UPDATE and WHERE line so that they are indented less. That maps to how we try to lay out SQL for better readability elsewhere.